core/audits/no-unload-listeners.js

```diff
@@ -14,7 +14,7 @@ const UIStrings = {
   /** Descriptive title of a Lighthouse audit that checks if a web page has 'unload' event listeners and finds that it is using them. */
   failureTitle: 'Registers an `unload` listener',
   /** Description of a Lighthouse audit that tells the user why pages should not use the 'unload' event. This is displayed after a user expands the section to see more. The last sentence starting with 'Learn' becomes link text to additional documentation. */
-  description: 'The `unload` event does not fire reliably and listening for it can prevent browser optimizations like the Back-Forward Cache. Use `pagehide` or `visibilitychange` events instead. [Learn more about unload event listeners](https://web.dev/bfcache/#never-use-the-unload-event)',
+  description: 'The `unload` event does not fire reliably and listening for it can prevent browser optimizations like the Back-Forward Cache. Use `pagehide` or `visibilitychange` events instead. [Learn more about unload event listeners](https://web.dev/articles/bfcache#never_use_the_unload_event)',
```

Comment:
@tunetheweb maybe fragments that appear in audit descriptions are good candidates for the manually-added anchors on web.dev? These will be updated for future reports, but all the existing reports out there will have the old broken fragments in there

Reply:
Good idea as there's not many of them. Will open a CL.

Reply:
Opened cl/572969626 and tagged you in it for review.